frontend/Dockerfile.frontend

```diff
@@ -0,0 +1,19 @@
+# Uses a multi-stage container build to build the frontend.
+#
+ARG REGISTRY
+FROM ${REGISTRY}/ubi8/go-toolset:1.20.10 AS builder
+
+USER root
+ENV GOPATH=/root/go
+RUN mkdir -p /app
+WORKDIR /app
+
+COPY . /app
+RUN make frontend
+
+FROM ${REGISTRY}/ubi8/ubi-minimal
+RUN microdnf update && microdnf clean all
+COPY --from=builder /app/aro /usr/local/bin/
+ENTRYPOINT ["aro"]
+EXPOSE 2222/tcp 8080/tcp 8443/tcp 8444/tcp 8445/tcp
```

Comment:
I copied the file from aro-rp and these are their ports for their lb, health probe, and testing. I'll change these as i understand better how our frontend will look like

Reply:
It would be nice to document this as a comment.
Even set it as variables and/org build variables with defaults if we don't mind about the extra layers